### DISCUSSION TITLE
Path of Wild Magic Barb?

### DISCUSSION THREAD

**Discussion by aceofkittenspace** [johnnolan/wild-magic-surge-5e#486] [Q&A]
I grabbed this module as I have a wild magic barb in my group, but I must admit I'm at a loss as to how to get it to trigger. There is no setup request for the Path, even though the Path is listed as an option. Do I need to add for some reason the Wild Magic Surge feats even if the Barb is not a Sorc for the module to work? All there is is statements what the module should do, but no set up for the Barbarian. Any explanation/help would be appreciated.

**Comment by johnnolan** [comment]
Hi, thanks for raising. Will leave some instructions here and also later in the week update the docs specifically with these instructions.

Ensure you have the subclass `Path of Wild Magic` set on the character like this screen shot.

![image](https://user-images.githubusercontent.com/5390820/220276026-759a85ee-45a4-40df-95b1-c6f904ff3bd9.png)

Then in the module settings, ensure you have `Auto Roll Table` set as so. I am going to do a patch soon to enable it so you can use the option for the player to roll from the message.

![image](https://user-images.githubusercontent.com/5390820/220276392-9d2a0571-99a2-4a58-a838-3fbd3afe6c5d.png)

If you don't want the players to see the roll messages you can set the following in settings so that only you can see it

![image](https://user-images.githubusercontent.com/5390820/220276893-48a56a61-5289-4b2a-a2ad-f91beff7f1d1.png)

If its still not working for you, could you create an issue for me and export your character and add it to the ticket and I can look at what the issue may be, thanks!

**Comment by johnnolan** [comment]
Documentation updated in README here https://github.com/johnnolan/wild-magic-surge-5e/blob/main/docs/POWN_BARB.md
